es-space-mgmt/main/default/classes/ReservationManagerController.cls

```diff
@@ -74,4 +74,61 @@ public with sharing class reservationManagerController {
             LIMIT 10
         ];
     }
+
+    @InvocableMethod
+    public static List<Id> updateReservationStatus(
```

Comment:
Invocable method to update the sobject record using custom metadata entries.

Reply:
Add a label to the annotation and move the invocable method to its own file as this is not part of the controller.

Reply:
done!

Reply:
This is done! Please review